hoot-core/src/main/cpp/hoot/core/schema/OsmSchema.cpp

```diff
@@ -1793,6 +1830,54 @@ double OsmSchema::score(const QString& kvp, const Tags& tags)
   return maxScore;
 }
 
+double OsmSchema::scoreTypes(const Tags& tags1, const Tags& tags2)
```

Comment:
Custom tag type scoring routine used in the generic geometry scripts, similar to what POI/Poly uses. Its possible this could be derived more simply from an existing `OsmSchema` method but I wasn't able to do so yet, so added this one. POI/Poly needs to be updated to use this, if possible.